documents/motions/21.2021.01 - WCA Documents.md

```diff
@@ -0,0 +1,27 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 21.2021.1  
+**Subject:** WCA Documents  
+**Intent:** Establish priorities and amendment procedures of WCA Documents  
+**Submitted by:** Motion has been removed and all responsibilities will be absorbed by the WCA Results Team.  
+**Date:** February 28, 2021  
+
+# Motion
+
+1. In case of any dispute between two WCA Documents the following order of priority shall be established:
+  1. WCA Bylaws
+  2. WCA Motions
+  3. WCA Regulations and Guidelines
+  4. WCA Policies
+  5. Other WCA material
+2. In case of any dispute between two WCA Documents with the same priority, the WCA Board shall determine which document takes precedence.
+3. In case of any dispute between two translations of the same document the American English version shall prevail
+4. Amendments of WCA Documents:
+  1. WCA Bylaws shall be amended by the procedures set out in section 12 of the Bylaws.
+  2. WCA Motions shall be amended by the procedures set out in Motion 13.2020.1 Amendments of Motions.
+  3. WCA Regulations and Guidelines shall be amended by the procedures set out in Motion 14.2020.1 Amendments of Regulations.
+  4. WCA Policies shall be amended by a majority vote among the WCA Board.
+  5. Other WCA Material shall be amended as necessary with approval of the WCA Board.
```

Comment:
I think this depends on what the WCA Board considers as "Other WCA Material". Does this include the software (e.g., TNoodle)? Because, in this case, the Amendment only depends on the WST and the WRC.

If this only refers to documents and not software. Does this include the WCA Educational Documents? https://www.worldcubeassociation.org/education

Reply:
T-noodle is not a document it is a software. Yes this would include the Educational documents. Given the wording of the proposed motion the Board is able to provide approval to a committee/group of committee's to amend other WCA material as required.